### PR TITLE
Feature/all cocktails

### PR DESCRIPTION
**Pull request by Melizzo**
# PR Description
Please include a summary of the change:
Got the data from API for all cocktails, and completed state/useeffect to store data upon render of cocktails page

Commit Emojis:
:tada: Initial commit
:crystal_ball: JS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Iteration 1

Note any questions or struggles as well.
We struggled and figured out how to correctly set up the interface of an array of objects.  

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import { getAllCocktails } from "../apiCalls";
+
+export interface AllCocktailsDetails {
+  drinks: [
```
Comment (Melizzo) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:5]: Léah and I figured out this interface and got our data to load correctly!

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import { getAllCocktails } from "../apiCalls";
+
+export interface AllCocktailsDetails {
```
Comment (Melizzo) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:4]: Had to update page as AllCocktails was being used as the state for an array of cocktails - a better defined component name.

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    {
+      strDrink: string;
+      strDrinkThumb: string;
+      idDrink: string;
+    }
+  ];
+}
+
+const AllCocktailsPage: React.SFC = () => {
+  const [allCocktails, setAllCocktails] = useState<AllCocktailsDetails>({
```
Comment (asodemann18) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:15]: Looks good!

File: src/apiCalls.tsx [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
+import { AllCocktailsDetails } from "./AllCocktailsPage/AllCocktailsPage";
+
 const rootUrl = "https://www.thecocktaildb.com/api/json/v1/1";
 
+//random cocktail: https://www.thecocktaildb.com/api/json/v1/1/random.php
+
+//lookup by id: https://www.thecocktaildb.com/api/json/v1/1/lookup.php?i=11007
+
+//Add AlcoholicCocktail[] to definitions
+export const getAllCocktails = async (): Promise<AllCocktailsDetails> => {
```
Comment (asodemann18) [inline src/apiCalls.tsx:11]: Nice job typing the function!

**Review by asodemann18 — APPROVED** [review]
Looks great!

File: src/AllCocktailsPage/AllCocktailsPage.tsx (hunk 2 above)
Comment (st0nebreaker) [reply src/AllCocktailsPage/AllCocktailsPage.tsx:4]: Yes. Good call! More semantic naming convention @Melizzo @LeahWinters

File: src/apiCalls.tsx (hunk 4 above)
Comment (st0nebreaker) [reply src/apiCalls.tsx:11]: 😎  some suave lookin code @Melizzo @LeahWinters

**Pull request merged**